controllers/binder_test.go

```diff
@@ -267,45 +267,40 @@ func TestBinderNew(t *testing.T) {
 	})
 
 	t.Run("remove", func(t *testing.T) {
+		fakeMock := mocks.NewFake(t, ns)
```

Comment:
This unit test tests the volume related piece of code added above -> https://github.com/redhat-developer/service-binding-operator/pull/907/files#r599403982